### ISSUE TITLE
Full path in traceback in built app

### ISSUE THREAD

**Issue by kosmos342** [pyinstaller/pyinstaller#2494]
There is full path from built machine in traceback.
Checked with Pyinstaller 3.2.1 and develop on macos.

How to reproduce:
create file **test.py** with content:
`import test2`

create file **test2.py** with content:
`assert False`

build:
`pyinstaller test.py`

run:
`./dist/test/test`

And in output we can see full path of pyinstaller from build machine. In my example it is /usr/local/var/pyenv/versions/3.5.3/envs/sa/lib/python3.5/site-packages/PyInstaller/loader/pyimod03_importers.py

output:
```
Traceback (most recent call last):
  File "test.py", line 1, in <module>
    import test2
  File "<frozen importlib._bootstrap>", line 969, in _find_and_load
  File "<frozen importlib._bootstrap>", line 958, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 673, in _load_unlocked
  File "/usr/local/var/pyenv/versions/3.5.3/envs/sa/lib/python3.5/site-packages/PyInstaller/loader/pyimod03_importers.py", line 631, in exec_module
    exec(bytecode, module.__dict__)
  File "test2.py", line 1, in <module>
    assert False
AssertionError
Failed to execute script test
```

**Comment by xoviat** [comment]
What did you expect? `assert False` would not allow the app to succeed.

**Comment by kosmos342** [comment]
I expect to not show full path "/usr/local/var/pyenv/versions/3.5.3/envs/sa/lib/python3.5/site-packages/PyInstaller/loader/pyimod03_importers.py"

**Comment by legendar** [comment]
@xoviat package builds on one machine, and then runs on another. stack trace contain full path from machine where builds was done.

**Comment by htgoebel** [comment]
Looks like stripping the path #1724 does not work for the modules imported by the boot-scripts.
Entry-points for debugging:
```
git grep pyimod03_importers PyInstaller
git grep strip_paths_in_code PyInstaller
```

**Comment by bwoodsend** [comment]
I can confirm that this is fixed by #5380.
```bash
dist/test/test
Traceback (most recent call last):
  File "test1.py", line 1, in <module>
    import test2 
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 359, in exec_module
  File "test2.py", line 1, in <module>
    assert 0 
AssertionError
[7816] Failed to execute script test1
```
